python/hsfs/core/feature_group_base_engine.py

```diff
@@ -61,6 +61,22 @@ def get_parent_feature_groups(self, feature_group):
             `ProvenanceLinks`:  the feature groups used to generate this feature group
         """
         return self._feature_group_api.get_parent_feature_groups(feature_group)
+    
+    def get_parent_storage_connectors(self, feature_group):
```

Comment:
This is mainly for my own understanding, can a feature group have multiple storage connector parents?